index.html

```diff
@@ -15,43 +15,46 @@
       <div class="status-bars">
       </div>
       <div class="app-bar">
-        <div class="toolbar">
+        <div class="toolbar-mobile">
+          <div class="logo">
+            <span>My Logo</span>
+          </div>
           <div class="normal-button">
-            <img class="burger" src="assets/burger.svg" alt="Menu icon.">
-            </div>
-            <div class="logo">
-              <span>My Logo</span>
-            </div>
-            <div class="toolbar-container">
-            </div>
+            <img class="burger" src="assets/burger.svg" alt="Menu icon">
           </div>
         </div>
+        <div class="toolbar-desktop">
+          <div class="logo">
+            <span>My Logo</span>
+          </div>
+          <nav>
+            <ul class="menu">
+              <li><a href="#portfolio">Portfolio</a></li>
+              <li><a href="#about">About</a></li>
+              <li><a href="#contact">Contact</a></li>
+            </ul>
```

Comment:
- Please remove the underline on the nav links `Portfolio, About, Contact`, so that it looks like the Figma design. 

##### Figma Design
![Screenshot from 2022-06-03 18-21-48](https://user-images.githubusercontent.com/58771507/171914688-7dbbfb11-7c40-488f-ab34-52b4cdf41216.png)

##### Your Design
![Screenshot from 2022-06-03 18-19-45](https://user-images.githubusercontent.com/58771507/171914744-b05feaea-7ef7-430c-8835-b27f1f7f435b.png)